kore/test/Test/Kore/Exec.hs

```diff
@@ -1227,6 +1228,7 @@ execTest depthLimit breadthLimit verifiedModule strategy initial = do
     exec
         depthLimit
         breadthLimit
+        Leaf
```

Comment:
Might be good to also add a unit test for execution to a branch point. It would be easy to add one in the style of `test_execBranch`.

Reply:
Yes, I don't think it's necessary either, feel free to merge.

Reply:
Yes, OK to merge... just that I am not running the integration tests all the time, but tend to use unit tests on every file save. For this particular feature it is probably fine not to test with a unit test.